client/components/Planning/PlanningItem.tsx

```diff
@@ -29,6 +29,7 @@ import {
     getItemWorkflowState,
 } from '../../utils';
 import {renderFields} from '../fields';
+import {gettext} from 'superdesk-core/scripts/core/utils';
```

Comment:
It would be better to use the `superdeskApi` instead of importing from `superdesk-core` directly